src/NCPopupTextEntry.h

```diff
@@ -58,6 +58,12 @@ class NCPopupTextEntry : public NCPopup
     void   setValue( const std::string & ntext ) { wtext->setValue( ntext ); }
 
     std::string value() { return wtext->value(); }
+
+    static std::string askForText( const wpos at,
+                                   const std::string & label,
+                                   const std::string & text,
```

Comment:
maybe make text default param `""`? It looks for me as usual case, where you ask for something.

Reply:
Yes, we can do that.